src/lib/api/builder.ts

```diff
@@ -666,8 +640,21 @@ export class BuilderAPI extends BaseAPI {
   }
 
   async fetchWeeklyStats(base: string) {
-    const remoteStats: RemoteWeeklyStats = await this.request('get', `/analytics/weekly?base=${base}`)
-    return fromRemoteWeeklyStats(remoteStats)
+    if (!statsPromise) {
+      statsPromise = fetch('https://cdn-data.decentraland.org/scenes/scene-stats.json').then(resp => resp.json())
+    }
+    const json = await statsPromise
+    const stats = base in json ? json[base]['last_7d'] : null
+
+    const weeklyStats: WeeklyStats = {
+      base,
+      users: stats ? stats.users : 0,
+      sessions: stats ? stats.sessions : 0,
+      medianSessionTime: stats ? stats.median_session_time : 0,
+      maxConcurrentUsers: stats ? stats.max_concurrent_users || 0 : 0
```

Comment:
This is a suggestion, but this might look better with the optional operators:
```suggestion
      users: stats?.users ?? 0,
      sessions: stats?.sessions ?? 0,
      medianSessionTime: stats?.median_session_time ?? 0,
      maxConcurrentUsers: stats?.max_concurrent_users ?? 0
```